### ISSUE TITLE
Review color accessibility

### ISSUE THREAD

**Issue by xak** [SAP/fundamental#96]
It seems some of our colors are not passing AA WCAG. Please double check.
https://sap.github.io/fundamental/colors.html

**Comment by joseegm** [comment]
@xak
Have you more details? Manu could chime in.
Some color combinations of the whole color pallete will not pass but those combinations should not be in use, as far as I know.

The text color/background color combination used should be mostly this ones.
<img width="1305" alt="screen shot 2018-02-27 at 10 14 05" src="https://user-images.githubusercontent.com/22662903/36720008-fa93caf0-1ba6-11e8-8fc1-74d37c9f7d87.png">

@LeoT7508 
Let's wait to get some more details on this from Manu,

**Comment by LeoT7508** [comment]
@joseegm 

I'll look into this when I get into the office.

**Comment by xak** [comment]
Neither action color passes. I did not test them all.

![screen shot 2018-02-27 at 8 51 37 am](https://user-images.githubusercontent.com/424119/36735336-8d7aa08e-1b9b-11e8-8ad6-aae369f50422.png)

![screen shot 2018-02-27 at 8 54 19 am](https://user-images.githubusercontent.com/424119/36735451-dfaa5174-1b9b-11e8-950c-f222fb58d3c7.png)

**Comment by joseegm** [comment]
Buffffff. At this point in the conversation last year i though we were all safe
https://github.com/SAP/techne/issues/757#issuecomment-310975617

**Comment by jeannewalters** [comment]
I was just noticing on the cloud commerce invision screens, their colors are a bit darker blues and lighter grays. https://projects.invisionapp.com/share/Q3FEPPRD57V#/screens/274270862

**Comment by saadmhybris** [comment]
The Teal on white we are using on the documentation site is also failing. 

<img width="721" alt="site-color-fail" src="https://user-images.githubusercontent.com/15215400/36736833-25358b8e-1b9f-11e8-821d-8c848defe118.png">

**Comment by LeoT7508** [comment]
@joseegm 

This was all addressed in the email I sent out, I will close the issue.

**Comment by joseegm** [comment]
This is still not finished

**Comment by LeoT7508** [comment]
@joseegm 

This is finished.
